tests/benchmarks/app_kernel/testcase.yaml

```diff
@@ -6,7 +8,7 @@ tests:
     tags: benchmark
     slow: true
     timeout: 300
-  test_posix:
+  benchmark.application.posix:
```

Comment:
These changes don't look related to the script changes. Should they be in the "tests: fix test identifiers" commit instead?